.github/workflows/update-changelog.yaml

```diff
@@ -0,0 +1,96 @@
+name: Update changelog
+
+concurrency:
+  group: update-changelog-${{ github.event_name }}-${{ github.ref }}
```

Comment:
Q: if `v` and `e` tags are pushed concurrently, maybe this could make one cancel the other?

Reply:
We don't usually do this now days, `e` tag is pushed separately to the private fork.